src/Exceptions/HttpGatewayTimeout.php

```diff
@@ -8,4 +8,3 @@ class HttpGatewayTimeout extends HttpException {
     public const DESCRIPTION = "Indicates that the server, while acting as a gateway or proxy, did not receive a timely response from an upstream server it needed to access in order to complete the request.";
     public const HREF = "https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/504";
 }
-
```

Comment:
Removed unix line ending - ALWAYS have a unix line ending, it can cause very weird issues.

Just use https://marketplace.visualstudio.com/items?itemName=jkiviluoto.tws you can use this to add them automatically. It also removes trailing whitespaces